### PR TITLE
GLTFLoader: Replace for...in with classic for loops on arrays.

### PR DESCRIPTION
**Pull request by marcatec**
Some for..in statements lurked in after #12173.

### CODE REVIEW

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -2242,13 +2242,13 @@ THREE.GLTFLoader = ( function () {
 
 		// Nothing in the node definition indicates whether it is a Bone or an
 		// Object3D. Use the skins' joint references to mark bones.
-		for ( var skinIndex in skins ) {
+		for ( var i =  0; i < skins.length; i ++ ) {
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: Could we keep the variable name as `skinIndex` here, and `nodeIndex` below? Given the multiple and nested `for` loops in this method, I think those names are more readable.

Comment (marcatec) [reply examples/js/loaders/GLTFLoader.js]: That's no problem.

**Review by donmccurdy — APPROVED** [review]
Looks good, thanks!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**